zetaclient/hsm_signer.go

```diff
@@ -0,0 +1,167 @@
+package zetaclient
```

Comment:
I think we could create a sub package `hsm` in ZetaClient `zetaclient/hsm/...` for better organization since `zetaclient` is getting bigger